app/src/main/java/com/example/androidtrainingproject/ui/home/HomeScreen.kt

```diff
@@ -162,6 +229,136 @@ fun Home(navController: NavController) {
                     }
                 }
             }
+
+            if (sheetState.isVisible) {
+                ModalBottomSheet(
+                    modifier = Modifier.fillMaxSize(),
+                    onDismissRequest = {
+                        coroutineScope.launch {
+                            sheetState.hide()
+                        }
+                    },
+                    sheetState = sheetState
+                ) {
+                    Column(
+                        modifier = Modifier.padding(horizontal = dimensionResource(id = R.dimen.padding_l))
+                    ) {
+                        Row {
+                            Icon(Icons.Outlined.Close, contentDescription = null,
+                                modifier = Modifier.clickable {
+                                    homeViewModel.removeAllFilters()
+                                    coroutineScope.launch {
+                                        sheetState.hide()
+                                    }
+                                })
+                            Text(
+                                text = stringResource(id = R.string.filters),
+                                modifier = Modifier.padding(start = dimensionResource(id = R.dimen.padding_sm)),
+                                style = MaterialTheme.typography.titleSmall.copy(
+                                    color = Black
+                                )
+                            )
+                            Spacer(modifier = Modifier.weight(1f))
+                            Text(
+                                text = stringResource(id = R.string.save),
+                                style = MaterialTheme.typography.titleSmall.copy(
+                                    color = Purple
+                                ),
+                                modifier = Modifier.clickable {
+                                    homeViewModel.applyFilters(
+                                        checkedCategories,
+                                        ratingFilter,
+                                        priceRange
+                                    )
+
+                                    coroutineScope.launch {
+                                        sheetState.hide()
+                                    }
+                                }
+                            )
+                        }
+                        CategoriesAccordion(
+                            homeViewModel = homeViewModel,
+                            checkedCategories = checkedCategories,
+                            ::checkCategory,
```

Comment:
Always name your parameters, except if there's not just only one or two of them and it's more than obvious what they are.